data/json/items/resources/misc.json

```
@@ -220,5 +220,19 @@
     "flags": [ "SPEAR", "SHEATH_KNIFE" ],
     "price": 1000,
     "price_postapoc": 0
+  },
+  {
+    "type": "GENERIC",
+    "id": "cannonball_4lb",
+    "symbol": "o",
+    "color": "dark_gray",
+    "name": { "str": "cannonball" },
+    "description": "A 4 pound cannonball. With some gunpowder, wadding, and a fuse, it could be prepared to load into a cannon.",
+    "material": [ "steel" ],
+    "weight": "1814 g",
+    "volume": "2000 ml",
+    "bashing": 15,
+    "to_hit": -3,
```

```suggestion
    "to_hit": -5,
```
Accuracy calculation:
-2 to start.
grip is bad (-1)
length is none (0)
striking surface is any (+0)
balance is clumsy (-2) 

Total is -5.